crates/sui-types/src/storage/object_storage_config.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Can this be moved into the sui-config crate instead of the sui-types crate?

Then sui-storage can depend on the config crate.